File: 01-iniciante/1020/1020.c
#include <stdio.h>

int main()
{
    int n,i;
    int ano = 0;
    int mes = 0;
    int dia = 0 ;

    scanf("%d",&n); 
    int valorAux = n;
    
    while(valorAux > 0) {
        if (valorAux >= 365){
            ano ++;
            valorAux -= 365;
        }
        else if (valorAux>= 30){
            mes ++;
            valorAux -= 30;
        }
        else {
            dia = valorAux;
            break;
        } 
    }
   
   printf("%d ano(s)\n",ano);
   printf("%d mes(es)\n",mes);
   printf("%d dia(s)\n",dia);
   
   return 0;
}

File: 01-iniciante/1020/1020.py
n = int(input())
valorAux = n 

ano = mes = dia = 0

while valorAux >= 0:
    if valorAux >= 365:
        ano += 1
        valorAux -= 365
    elif valorAux >= 30:
        mes += 1
        valorAux -= 30
    else:
        dia = valorAux
        break
        
print(f"{ano} ano(s)")
print(f"{mes} mes(es)")
print(f"{dia} dia(s)")